acceptance-tests/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
as discussed, perhaps this could run npm install? (maybe best to let user run it?)